### PR TITLE
Add doc comments to main

### PR DESCRIPTION
**Pull request by joepetrowski**
Probably should use something like typedoc, but thought I'd get started with this. We should also consider adding `/docs/` endpoints that would fetch the docs from the metadata on anything returned.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -71,19 +71,87 @@ async function main() {
...
+	// GET balance information for an address.
+	//
+	// Paths:
+	// - `address`: The address to query.
+	// - (Optional) `number`: Block number at which to query. If not provided, queries finalized head.
+	//
+	// Returns:
+	// - `at`: Block number and hash at which the call was made.
+	// - `nonce`: Account nonce.
+	// - `free`: Free balance of the account. Not equivalent to _spendable_ balance. This is the only
```
Comment (emostov) [inline src/main.ts:138]: Not necessary, but I think the docs could be more comprehensive by linking to something like this for balances: https://stackoverflow.com/questions/56418100/what-is-the-difference-between-free-balance-reserved-balance-locked-balance-v

File: src/main.ts [hunk 2]
```diff
@@ -99,6 +167,22 @@ async function main() {
...
+	// Paths:
+	// - `address`: The _Stash_ address for staking.
+	// - (Optional) `number`: Block number at which to query. If not provided, queries finalized head.
+	//
+	// Returns:
+	// - `at`: Block number and hash at which the call was made.
+	// - `rewardDestination`: The account to which rewards will be paid. Can be 'Staked' (Stash
+	//   account, adding to the amount at stake), 'Stash' (Stash address, not adding to the amount at
+	//   stake), or 'Controller' (Controller address).
+	// - `bonded`: Controller address for the given Stash.
```
Comment (emostov) [inline src/main.ts:181]: Should this mention the controller address can also possibly be the stash address? I think that can get a little confusing.

File: src/main.ts [hunk 3]
```diff
@@ -114,6 +198,32 @@ async function main() {
...
 
+	// GET staking information for an address.
+	//
+	// Paths:
+	// - `address`: The _Controller_ address for staking.
+	// - (Optional) `number`: Block number at which to query. If not provided, queries finalized head.
+	//
+	// Returns:
+	// - `at`: Block number and hash at which the call was made.
+	// - `stash`: The stash account whose balance is actually locked and at stake.
```
Comment (emostov) [inline src/main.ts:209]: If we want to address my above comment we could add the vice versa here; Stash could be the same as controller

File: src/main.ts (hunk 1 above)
Comment (joepetrowski) [reply src/main.ts:138]: I don't think so, that SO isn't maintained and these should be strictly reference.

File: src/main.ts (hunk 2 above)
Comment (joepetrowski) [reply src/main.ts:181]: Maybe I'll just link to the ref doc of the Staking pallet. But I think this should just say what this function does, not try to explain all of substrate.

**Pull request merged**